webpack.prod.js

```diff
@@ -0,0 +1,19 @@
+const merge = require('webpack-merge');
```

Comment:
como esse merge trata campos que são objeto ou array?

Reply:
Segundo o [primeiro parágrafo](https://github.com/survivejs/webpack-merge#webpack-merge---merge-designed-for-webpack) do readme da lib: "webpack-merge provides a merge function that concatenates arrays and merges objects creating a new object. If functions are encountered, it will execute them, run the results through the algorithm, and then wrap the returned values within a function again."